File: annotation-processor/src/test/java/me/bristermitten/mittenlib/annotations/util/PrivateAnnotationsTest.java
package me.bristermitten.mittenlib.annotations.util;

import static org.junit.jupiter.api.Assertions.*;

import me.bristermitten.mittenlib.config.names.ConfigName;
import org.junit.jupiter.api.Test;

class PrivateAnnotationsTest {

    @Test
    void testIsPrivate() {
        assertTrue(PrivateAnnotations.isPrivate(ConfigName.class.getName()));
        assertFalse(PrivateAnnotations.isPrivate("some.other.Annotation"));
    }
}

File: annotation-processor/src/test/scala/me/bristermitten/mittenlib/annotations/util/PrivateAnnotationsTest.scala
package me.bristermitten.mittenlib.annotations.util

import me.bristermitten.mittenlib.config.names.ConfigName
import org.scalatest.funsuite.AnyFunSuite
import org.scalatest.matchers.should.Matchers

class PrivateAnnotationsTest extends AnyFunSuite with Matchers {

  test("testIsPrivate") {
    PrivateAnnotations.isPrivate(classOf[ConfigName].getName) shouldBe true
    PrivateAnnotations.isPrivate("some.other.Annotation") shouldBe false
  }
}
